apps/chat/src/store/application-type-schemas/application-type-schemas.selectors.ts

```diff
@@ -0,0 +1,17 @@
+import { createSelector } from '@reduxjs/toolkit';
+
+import { RootState } from '../index';
+import { ApplicationTypesSchemasState } from './application-type-schemas.reducer';
+
+const rootSelector = (state: RootState): ApplicationTypesSchemasState =>
+  state.applicationTypesSchemas;
+
+export const selectAllSchemas = createSelector(
+  [rootSelector],
+  (state) => state.schemas,
+);
```

Comment:
`createSelector` doesn't make sense here
```suggestion
export const selectAllSchemas = (state: RootState) =>
  rootSelector(state).schemas;
```

Reply:
done